tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py

```diff
@@ -16,140 +16,397 @@
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.isInLimits(0, 5)
+                    ]),
+              Field(item="34", name='RELATIONSHIP_HOH', type='string', startIndex=51, endIndex=53,
+                    required=True, validators=[
+                        validators.isInStringRange(1, 10)
+                    ]),
+
+              # naming difference between t2?
               Field(item="35", name='PARENT_MINOR_CHILD', type='number', startIndex=53, endIndex=54,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.isInLimits(0, 3)
+                    ]),
               Field(item="36", name='NEEDS_PREGNANT_WOMAN', type='number', startIndex=54, endIndex=55,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.isInLimits(0, 9)
+                    ]),
               Field(item="37", name='EDUCATION_LEVEL', type='number', startIndex=55, endIndex=57,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.or_validators(
+                            #   validators.isInStringRange(0, 16),
+                            validators.isInLimits(0, 16),
+                            #   validators.isInStringRange(98, 99)
+                            validators.isInLimits(98, 99)
+                        )
+                    ]),
               Field(item="38", name='CITIZENSHIP_STATUS', type='number', startIndex=57, endIndex=58,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.oneOf([0, 1, 2, 3, 9])  # option 3 missing from t2
+                    ]),
               Field(item="39", name='COOPERATION_CHILD_SUPPORT', type='number', startIndex=58, endIndex=59,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.oneOf([0, 1, 2, 9])
+                    ]),
               Field(item="40", name='EMPLOYMENT_STATUS', type='number', startIndex=59, endIndex=60,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.isInLimits(0, 3)
+                    ]),
               Field(item="41", name='WORK_ELIGIBLE_INDICATOR', type='number', startIndex=60, endIndex=62,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.or_validators(
+                            #   validators.isInStringRange(0, 9),
+                            validators.isInLimits(0, 9),
+                            #   validators.matches('12') # missing 11 from t2
+                            validators.matches(12)
```

Comment:
a number of fields (especially in `m2`, though a few cases exist in `m1` and `m3` as well) are represented as strings in the corresponding TANF schema (`t1`, `t2`, `t3`), but are represented as numbers here, thus the validators needed to be adjusted to reflect the new data type. i left the old validators commented out to call out the discrepancies.

there are other differences between the SSP schema and the corresponding TANF schemas, as is noted in the comment `# missing 11 from t2` - validators themselves allow slightly different values. in some cases, that's reflected by new versions of the coding instructions. in some cases, i'm not sure. i left comments so that these areas could be looked at more closely.